### PR TITLE
[test](jdbc) add doris and sqlserver jdbc catalog test case

### PR DESCRIPTION
**Pull request by zy-kkk**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zy-kkk** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

File: regression-test/suites/external_table_p0/jdbc/test_doris_jdbc_catalog.groovy [hunk 1]
```diff
@@ -167,6 +171,12 @@ suite("test_doris_jdbc_catalog", "p0,external,doris,external_docker,external_doc
     order_qt_tb2 """ select pin_id, hll_union_agg(user_log_acct) from ${catalog_name}.${internal_db_name}.${hllTable} group by pin_id; """
     order_qt_base2 """ select * from ${catalog_name}.${internal_db_name}.${base_table} order by int_col; """
     order_qt_arr2 """ select * from ${catalog_name}.${internal_db_name}.${arr_table} order by int_col; """
+    sql """ drop table if exists internal.${internal_db_name}.ctas_base; """
+    sql """ drop table if exists internal.${internal_db_name}.ctas_arr; """
+    order_qt_ctas_base """ create table internal.${internal_db_name}.ctas_base PROPERTIES("replication_num" = "1") as select * from ${catalog_name}.${internal_db_name}.${base_table} order by int_col; """
```
Comment (BePPPower) [inline regression-test/suites/external_table_p0/jdbc/test_doris_jdbc_catalog.groovy:176]: we'd better add `desc table` test for table which is created by CTAS, to ensure the type mapping is correct

File: regression-test/suites/external_table_p0/jdbc/test_sqlserver_jdbc_catalog.groovy [hunk 2]
```diff
@@ -71,7 +71,10 @@ suite("test_sqlserver_jdbc_catalog", "p0,external,sqlserver,external_docker,exte
         order_qt_filter2  """ select * from test_char where 1 = 1 and id = 1  order by id; """
         order_qt_filter3  """ select * from test_char where id = 1  order by id; """
         order_qt_id """ select count(*) from (select * from t_id) as a; """
-
+        order_qt_all_type """ select * from all_type order by id; """
+        sql """ drop table if exists internal.${internal_db_name}.all_type; """
+        order_qt_ctas """ create table internal.${internal_db_name}.ctas_all_type PROPERTIES("replication_num" = "1") as select * from all_type; """
```
Comment (BePPPower) [inline regression-test/suites/external_table_p0/jdbc/test_sqlserver_jdbc_catalog.groovy:76]: we'd better add desc table test for table which is created by CTAS

**Comment by zy-kkk** [comment]
run buildall

**Review by BePPPower — APPROVED** [review]
LGTM

**Comment by zy-kkk** [comment]
run p0

**Comment by zy-kkk** [comment]
run buildall

**Review by JNSimba — APPROVED** [review]

**Pull request merged**